Library/Formula/librdkafka.rb

```diff
@@ -0,0 +1,12 @@
+class Librdkafka < Formula
+  homepage "https://github.com/edenhill/librdkafka"
+  url "http://github.com/edenhill/librdkafka/tarball/0.8.4"
```

Comment:
No good reason, they both appear to give the same result. Happy to change it over if that's preferred.